docs/action-from-pipeline-task.md

```diff
@@ -0,0 +1,48 @@
+| tl.getEndpoint()     | Take an input, whose expected value is produced using  `az sp ad create-for-rbac --sdk-auth` | The concept of serviceConnections doesn’t exist in Actions world.                                                                                                          |
+| tl.debug()           | core.debug()                                                                                 | Import * as core from ‘@actions/core’                                                                                                                                      |
+| tl.getInput()        | core.getInput()                                                                              | Import * as core from ‘@actions/core’, if input is required. Use  core.getInput({“required”:true});                                                                        |
+| tl.getPathInput()    | core.getInput()                                                                              | Given that path input was a UI feature and workflows are exclusively yaml based, you should use core.getInput instead, reading the content as plain text                   |
+| tl.getBoolInput()    | core.getInput()  Convert to type later                                                       |                                                                                                                                                                            |
+
+The above list contains a few observed conversions to help you get started. You might also internally use certain modules built especially for tasks, when converting to action, validate if any of your dependencies aren’t bringing in any `vsts-task-lib` references; you could potentially run into unexpected problems during runtime.
+
+If any of your code logic is completely independent of tasks-lib and any dependencies from the common-npm-packages folder. You can use the file/module/class as it is.
+
+## Converting task.json to action.yml
+
+Refer to this document for more information: https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions 
+
+Fields which exist in task.json but not in actions.yml
+* ID 
+* friendlyName
+* helpUrl
+* helpMarkDown
+* Category
+* Visibility
+* Groups
+* Version
+* Demands
+* Groups
+* dataSourceBindings
+* instanceNameFormat
+* Messages
+
+You can ignore the above attributes while converting your task.json
+
+Please find some examples below - 
+
+| From | To |
+|-----|-----|
+|“inputs”: [{<br>        "name": "<name>",<br>        "type": "<any-type>",<br>        "label": "<label>",<br>        "required": false,<br>        "defaultValue": "<default>",<br>        "options": {<br>        ....options<br>        },<br>        "helpMarkDown": "<helpText>"<br><br>    }],|inputs:<br>  <name>:<br>      description: <description><br>      required: false<br>      default: ‘<default>’|
```

Comment:
enclose these inputs in ```json and outputs in ```yaml

Reply:
and outputs in ```yaml

Reply:
There is some issue with using those in the table cells. Will have a look at this later